packages/atlas/src/utils/subtitles.ts

```diff
@@ -0,0 +1,33 @@
+export const converSrtToVtt = async (srtFile: File): Promise<File> => {
+  const subtitlesString = await new Promise<string>((resolve, reject) => {
+    const reader = new FileReader()
+    const onLoadEnd = (event: ProgressEvent<FileReader>) => {
+      const result = event.target?.result
+      reader.removeEventListener('loadend', onLoadEnd)
+      if (typeof result === 'string') {
+        resolve(result)
+      }
```

Comment:
If somehow `result` will not be a string, this will fail silently, let's call `reject`